bubblewrap.c

```diff
@@ -1432,6 +1432,19 @@ print_version_and_exit (void)
   exit (0);
 }
 
+static int
+parse_fd_arg (const char *arg)
+{
+  char *endptr;
+  int the_fd;
+
+  the_fd = strtol (arg, &endptr, 10);
+  if (arg[0] == 0 || endptr[0] != 0 || the_fd < 0)
```

Comment:
This looks like it can still produce a return value less than zero.

```suggestion
  if (arg[0] == 0 || endptr[0] != 0 || the_fd < 0 || the_fd > INT_MAX)
```

Reply:
Never mind, didn’t notice it was assigned to an `int` variable.

Reply:
Although it can still create strange truncations so I think this + `long the_fd` is necessary after all.